src/main/java/uk/gov/hmcts/divorce/document/content/AosUndefendedResponseLetterTemplateContent.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.divorce.document.content;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.common.service.HoldingPeriodService;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.APPLICANT_1_ADDRESS;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.APPLICANT_1_FIRST_NAME;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.APPLICANT_1_LAST_NAME;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.CASE_REFERENCE;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.CONTACT_DIVORCE_EMAIL;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.CP_CASE_EMAIL;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.DIVORCE_OR_CIVIL_PARTNERSHIP_EMAIL;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.DIVORCE_OR_CIVIL_PARTNERSHIP_SERVICE_HEADER;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.DIVORCE_SERVICE;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.END_CP_SERVICE;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.ISSUE_DATE;
+import static uk.gov.hmcts.divorce.document.content.DocmosisTemplateConstants.RELATION;
+import static uk.gov.hmcts.divorce.notification.CommonContent.IS_DIVORCE;
+import static uk.gov.hmcts.divorce.notification.FormatUtil.DATE_TIME_FORMATTER;
+import static uk.gov.hmcts.divorce.notification.FormatUtil.formatId;
+
+@Component
+@Slf4j
+public class AosUndefendedResponseLetterTemplateContent {
+
+    @Autowired
+    private CommonContent commonContent;
+
+    @Autowired
+    private HoldingPeriodService holdingPeriodService;
+
+    private static final String DATE_TO_WAIT_UNTIL_APPLY_FOR_CO = "dateToWaitUntilApplyForCO";
+
+    public Map<String, Object> apply(final CaseData caseData,
+                                     final Long ccdCaseReference) {
+
+        log.info("For ccd case reference {} and type(divorce/dissolution) {} ", ccdCaseReference, caseData.getDivorceOrDissolution());
+
+        Map<String, Object> templateContent = new HashMap<>();
+        templateContent.put(APPLICANT_1_FIRST_NAME, caseData.getApplicant1().getFirstName());
```

Comment:
use Applicant.getFullName instead

Reply:
getFullName returns full name including middle name as well, and verified with Michael and we don't want middle name to be displayed even if it is present that's why set first name and last name separately.

Reply:
we had an issue raised saying it must be there, it's their full legal name